components/ConfirmPopup.js

```diff
@@ -0,0 +1,54 @@
+import styled, { css } from "styled-components";
+
+export default function ConfirmPopup({ onConfirm, onClose }) {
+  return (
+    <Wrapper onClick={(e) => e.stopPropagation()}>
+      <ConfirmQuestion>Do you want to notice new item? </ConfirmQuestion>
+      <ButtonsWrapper>
+        <Button variant="confirm" onClick={onConfirm}>
```

Comment:
Please add a type for every button. Maybe if you add type="button" you do not need `event.stopPropagation()` anymore as the default type of each button is submit. 🤔